<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Battle</class>
 <widget class="QDialog" name="Battle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="enemyNameLabel">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="abilityOneB">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>579</y>
     <width>167</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="abilityTwoB">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>579</y>
     <width>167</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="abilityThreeB">
   <property name="geometry">
    <rect>
     <x>417</x>
     <y>579</y>
     <width>167</width>
     <height>56</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="itemsB">
   <property name="geometry">
    <rect>
     <x>618</x>
     <y>577</y>
     <width>167</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fleeB">
   <property name="geometry">
    <rect>
     <x>618</x>
     <y>608</y>
     <width>167</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="enemyHpPB">
   <property name="geometry">
    <rect>
     <x>502</x>
     <y>56</y>
     <width>305</width>
     <height>39</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="hpPB">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>56</y>
     <width>305</width>
     <height>39</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="mpPb">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>104</y>
     <width>263</width>
     <height>19</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="useB">
   <property name="geometry">
    <rect>
     <x>618</x>
     <y>577</y>
     <width>167</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeB">
   <property name="geometry">
    <rect>
     <x>618</x>
     <y>608</y>
     <width>167</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QListWidget" name="bagLW">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>550</y>
     <width>569</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="professionLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>150</width>
     <height>300</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="enemyLabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>270</y>
     <width>341</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>bagLW</zorder>
  <zorder>enemyNameLabel</zorder>
  <zorder>abilityOneB</zorder>
  <zorder>abilityTwoB</zorder>
  <zorder>abilityThreeB</zorder>
  <zorder>itemsB</zorder>
  <zorder>fleeB</zorder>
  <zorder>enemyHpPB</zorder>
  <zorder>hpPB</zorder>
  <zorder>nameLabel</zorder>
  <zorder>mpPb</zorder>
  <zorder>useB</zorder>
  <zorder>closeB</zorder>
  <zorder>professionLabel</zorder>
  <zorder>enemyLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
